nixos/modules/services/web-apps/suwayomi-server.md

```diff
@@ -0,0 +1,108 @@
+# Suwayomi-Server {#module-services-suwayomi-server}
+
+A free and open source manga reader server that runs extensions built for Tachiyomi.
+
+## Basic usage {#module-services-suwayomi-server-basic-usage}
+
+By default, the module will execute Suwayomi-Server backend and web UI:
+
+```nix
+{ ... }:
+
+{
+  services.suwayomi-server = {
+    enable = true
```

Comment:
I think this might be the reason why it's currently failing?

```suggestion
    enable = true;
```

Reply:
Indeed I forgot this, but the error came from the fact that I forgot to add the package and the module to the list of all packages/modules \^\^'